src/components/illustration/illustration.tsx

```diff
@@ -4,6 +4,7 @@ import styled, { css } from 'styled-components';
 
 export enum IllustrationSize {
   NORMAL = 'normal', // 72px
+  MEDIUM = 'medium', // 80px
```

Comment:
@georgiee I noticed that illustrations with 80px often appear in the design. Therefore I have added another size, I hope that's okay

Reply:
Alright, good for now. Can you make a comment in figma because the 8px difference feels like a design inconsistency rather than a decision. We might want to revert this change then.